example_python/test/test_parameters.py

```diff
@@ -0,0 +1,7 @@
+import generate_parameter_module_example
+print(f'Imported file from: {generate_parameter_module_example.__file__}')
+print('OK 1')
+
+import generate_parameter_module_example.admittance_parameters  # noqa: E402
+
+print('OK 2')
```

Comment:
note this test is only there for illustrative purposes -- we can either remove it or clean it up